### PR TITLE
Fix quiz layout

### LINKED ISSUE #145: Quiz : answers too long
Screen Shot 2021-01-24 at 11 06 28

### PR DESCRIPTION
**Pull request by valentinperignon**
Fixes #145.

Screen Shot 2021-01-26 at 15 17 24

### CODE REVIEW

**Review by FrancoisPog — APPROVED** [review]
LGTM

File: client/src/components/quizz/Question.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const Question = ({ numero, text }) => {
+  return (
+    <div id="quiz-question">
+      <h2>Question {numero}</h2>
+      <h1>{text}</h1>
+    </div>
+  );
+};
+
+Question.protoTypes = {
```
Comment (nathanaelhoun) [inline client/src/components/quizz/Question.js:13]:
```suggestion
Question.propTypes = {
```

**Comment by FrancoisPog** [comment]
Do you handle the case where an answer is longer than the width of the window ?

**Review by nathanaelhoun — APPROVED** [review]
LGTM! (just a typo)

**Pull request merged**